lib/prometheus/client/registry.rb

```diff
@@ -83,6 +83,10 @@ def get(name)
       def metrics
         @metrics.values
       end
+
+      def clear
+        @metrics.clear
```

Comment:
`@mutex.synchronize` please  :). (if we're going with this code, which I'm not sure we are, so maybe ignore me for now)

And come to think of it... Can we also please synchronize the 3 methods above? I'm not entirely sure how I didn't see that! 🙈

Reply:
Follow-up: #201